opendmarc/opendmarc-arcseal.c

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
Please note in comments that these functions are needed to extract seal details for inclusion in the DMARC reporting.  It's not needed for the protocol.